src/build-csv.js

```diff
@@ -32,6 +33,8 @@ const main = async () => {
           await writeFile(csvPath, csv);
         } catch (err) {
           console.error(`Error: Excel ファイル ${excelPath} を CSV に変換できませんでした。`);
+          
+          fs.writeFileSync('error-file.txt', excelPath);
```

Comment:
async 関数の中なので、fs/promises が使えると思います。

```suggestion
          await writeFile('error-file.txt', excelPath);
```

Reply:
@phanect ありがとうございます。下の件と併せて修正しました!